docs/about/compatibility/openmp.md

```diff
@@ -280,6 +280,61 @@ to by “b” are in coarse-grain memory during and after the execution of the
 target region. This is accomplished in the OpenMP runtime library with calls to
 the ROCr runtime to set the pages pointed by “b” as coarse grain.
 
+#### Zero-copy behavior on MI300A and discrete GPUs
+
+The runtime behavior on MI300A and other discrete GPUs depends on the status of XNACK and the existence of the pragma `requires unified_shared_memory` as discussed in the following sections.
+
+##### Implicit zero-copy behavior on MI300A
+
+OpenMP provides a relaxed shared memory model. Map clauses provided in the source code indicate how data is used and copied to and from the GPU device for each target region. On GPUs that provide USM like the MI300A, these clauses are optional but facilitate portability to discrete memory GPUs. The [unified shared memory pragma](#unified-shared-memory-pragma) `requires unified_shared_memory` informs the compiler and runtime that the code is NOT portable to discrete memory GPUs, and must be compiled and executed on GPUs providing USM, such as MI300A. The MI300A is one of the several AMDGPUs with [XNACK capability](#xnack-capability).
+
+The following table lists the runtime behavior based on the unified shared memory pragma and XNACK specification:
+
+| MI300A | `requires unified_shared_memory` <br>NOT specified</br> | `requires unified_shared_memory` <br>specified</br> |
+| --- | --- | --- |
+| XNACK enabled | implicit zero-copy |  zero-copy |
+| XNACK disabled | copy |  runtime warning* |
+
+###### Host memory prefaulting in zero-copy modes
```

Comment:
Let's cut this section.

Reply:
@carlobertolli addressed your comments. Please review.